lib/cluster/base.rb

```diff
@@ -35,6 +36,11 @@ def self.external_storage?
       storage_config[:type] == 'external'
     end
 
+    def self.show_zadara_tasks?
+      stack_name = config.parsed[:stack][:name]
+      external_storage? && ! stack_name.match(/prod|prd/i)
```

Comment:
Why is 'prod|prd' in the stack name significant? Is this a way to prevent accidental hibernation of the prod cluster?

Reply:
Essentially, yes. I sent out an email last week, "oc-staging cluster + odcev test vpsa", that gives some background. This is how mh-opsworks has historically identified something that might be a production cluster.